### PR TITLE
 fix decimal translation error when infer subscription channel.

### PR DESCRIPTION
**Pull request by gyunt**
Decimal.__str__ print itself as fixed point expression only if exp >= -6.

### CODE REVIEW

File: pydaybit/utility.py [hunk 1]
```diff
@@ -15,3 +16,15 @@ def decorated(*args, **kwargs):
         return decorated
 
     return args_deco
+
+
+def to_str(d):
+    if isinstance(d, Decimal):
```
Comment (jinkyuyi) [inline pydaybit/utility.py]: Is there any reason not simply call `Decimal.__format__()`? I think several lines below can be replaced as following:
```python
return '{:f}'.format(d)
```

File: setup.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
     long_description = fh.read()
 
 setup(name='pydaybit',
-      version='0.0.3.3',
+      version='0.0.3.4',
```
Comment (jinkyuyi) [inline setup.py]: Please don't bump version in patch. It will be held by internal release management team seperately.

**Comment by jinkyuyi** [comment]
Oh, and I just enabled Circle CI to run builds for pull requests from forks.

File: pydaybit/utility.py (hunk 1 above)
Comment (gyunt) [reply pydaybit/utility.py]: You are right. I until don't know about the **f** option. 
FYI, the type check, _isinstance_ will be needed for other types, such as string. 
Thanks.

**Pull request closed without merging**